HW02_B_Mimic_starter.Rmd

```diff
@@ -82,15 +123,24 @@ data(mpg)
 #hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
 library(RColorBrewer)
 display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
+
 ```
 
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
+The above graph lets you see some colorblind friendly palettes. For the graph below, I used Set2.
 
 
 
 Now using the above mpg dataset, make this graph
 

```

Comment:
The figure looks great. The only thing I would change is the position of the points relative to the outiline of the boxplots. Here points are over the boxes, and they should be below. For that, you would need to 1) Change the order of "geom_boxplot" and "geom_jitter" by putting "geom_jitter" first 2) Add the argument "fill=NA" to "geom_boxplot" so the boxes are not filled and you can actually see the points that are below.